src/thing_objects.c

```diff
@@ -569,6 +569,18 @@ void destroy_object(struct Thing *thing)
     }
 }
 
+TbBool object_is_indestructable (const struct Thing* thing)
+{
+    //todo make this an object property. Then include the possibility to kill the other object types.
+    if (thing_is_invalid(thing))
+        return false;
```

Comment:
I don't see how an invalid thing would be destructible

Reply:
Well, if the thing does not exist, it's not an indestructible object. If it's not an object, it's not an indestructible object either.

Reply:
yeah guess this one depends on the comment below if you'd set it to destructable instead it would still be false